circle.yml

```diff
@@ -6,6 +6,8 @@ machine:
   # Install ceph to test rados driver & create pool
     - sudo -i ~/distribution/contrib/ceph/ci-setup.sh
     - ceph osd pool create docker-distribution 1
+  # Enable the mystical non-privileged ping socket
+    - sudo sysctl -w net.ipv4.ping_group_range="0 1000"
```

Comment:
Linux users _may_ need to use `sudo` or have a profile set up in whichever linux security engine their distro uses to permit the application to open this type of socket. I don't think there is any way to avoid this :disappointed: Do you think it's worth adding something to the docstring for this health checker for now?